RevenueCatUI/CustomerCenter/Views/CustomerCenterCompletionHandler.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterCompletionHandler.swift
+//
+//  Created by Cesar de la Vega on 6/7/24.
+
+import Foundation
+import SwiftUI
+
+final class CustomerCenterCompletionHandler: ObservableObject {
+
+    @Published
+    fileprivate(set) var customerCenterResult: CustomerCenterResult?
+
+    static func `default`() -> Self {
+        return .init()
+    }
+
+    func supportContacted() {
+        self.customerCenterResult = CustomerCenterResult(status: .contactSupport)
+    }
+
+}
+
+struct CustomerCenterResult: Equatable {
```

Comment:
If we follow my previous suggestion, I would move away from using the `Result` naming, since this may happen at any point, and even multiple times. I would only keep this naming if we want to return a single result after the customer center closes, but I think I prefer my suggested option. Lmk what you think!